Documentation/example-measures/example-measure-kds.json

```diff
@@ -177,9 +271,216 @@
+          },
+          "code": {
+            "coding": [
+              {
+                "system": "http://fhir-data-evaluator/strat/system",
+                "code": "icd10-code"
+              }
+            ]
+          },
+          "id": "strat-9"
+        },
+        {
+          "criteria": {
+            "language": "text/fhirpath",
+            "expression": "Patient.gender"
+          },
+          "code": {
+            "coding": [
+              {
+                "system": "http://fhir-data-evaluator/strat/system",
+                "code": "icd10-code"
+              }
+            ]
+          },
+          "id": "strat-10"
+        }
+      ]
+    },
+    {
+      "population": [
+        {
+          "code": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                "code": "initial-population"
+              }
+            ]
+          },
+          "criteria": {
+            "language": "text/x-fhir-query",
+            "expression": "MedicationAdministration?_profile=https://www.medizininformatik-initiative.de/fhir/core/modul-medikation/StructureDefinition/MedicationAdministration"
+          },
+          "id": "initial-population-identifier-4"
+        },
+        {
+          "code": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                "code": "measure-population"
+              }
+            ]
+          },
+          "criteria": {
+            "language": "text/fhirpath",
+            "expression": "MedicationAdministration"
+          },
+          "id": "measure-population-identifier-4"
+        },
+        {
+          "code": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                "code": "measure-observation"
+              }
+            ]
+          },
+          "criteria": {
+            "language": "text/fhirpath",
+            "expression": "MedicationAdministration.subject.reference"
+          },
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-aggregateMethod",
+              "valueCode": "unique-count"
+            },
+            {
+              "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-criteriaReference",
+              "valueString": "measure-population-identifier-4"
+            }
+          ],
+          "id": "measure-observation-identifier-4"
+        }
+      ],
+      "stratifier": [
+        {
+          "criteria": {
+            "language": "text/fhirpath",
+            "expression": "(MedicationAdministration.medication).code.coding"
+          },
+          "code": {
+            "coding": [
+              {
+                "system": "http://fhir-data-evaluator/strat/system",
+                "code": "icd10-code"
```

Comment:
@DeaDurro  - same as above - for the whole measure use codes which describe what is being queried.
apply logic as follows:
resourceName-field-type:
=> here:
medicationadministration-medication-code